02-oop-inheritance/js/script.js

```diff
@@ -0,0 +1,82 @@
+var ObserverList = function () {
+    this.observerList = [];
+};
+
+ObserverList.prototype = {
+
+    constructor: ObserverList,
+
+    subscribeObserver: function (observer) {
+        this.observerList.push(observer);
+    },
+
+    unsubscribeObserver: function (observer) {
+        this.observerList.splice(this.observerList.indexOf(observer), 1);
+    }
+};
+
+var Movie = function (title,year,observer) {
+    this.title = title;
+    this.year = year;
+    ObserverList.call(observer);
+};
+
+Movie.prototype = {
+
+    constructor: Movie,
+
+    get: function (attribute) {
+        return this[attribute];
+    },
+
+    set: function (attribute, value) {
+        this[attribute] =  value;
+    },
+
+    startplaying: function () {
+        for( i = 0; i < observerList.length ; i++) {
```

Comment:
**Minor:** add space after ``for`` declaration and remove spaces after ``length`` and the start of parenthesis (should be ``for (i = 0; i < observerList.length; i += 1)``).
Also, a few good practices tips: try to define ``i`` before the for loop and use ``i += 1`` instead ``i++``, something like this:
```
startplaying: function () {
    var i = 0;

    for (i; i < observerList.length; i+= 1) {
        // ...
    }
},
```